web/client/plugins/FeatureEditor.jsx

```diff
@@ -175,7 +183,9 @@ const EditorPlugin = connect(
                 virtualScroll: this.props.virtualScroll ?? true,
                 editingAllowedRoles: this.props.editingAllowedRoles,
                 editingAllowedGroups: this.props.editingAllowedGroups,
-                maxStoredPages: this.props.maxStoredPages
+                maxStoredPages: this.props.maxStoredPages,
+                restrictedAreaUrl: this.props.restrictedAreaUrl,
```

Comment:
this is not present in config, only restrictedArea with inside three params:
- raw
- operator
- url